contracts/DelFiPrice.sol

```diff
@@ -238,16 +239,16 @@ contract DelFiPrice is OpenOracleView {
      * @param cToken The cToken address to map to symbol
      * @return The symbol for the given cToken address
      */
-    function getOracleKey(address cToken) public view returns (string memory) {
-        if (cToken == cEthAddress) return "ETH";
-        if (cToken == cUsdcAddress) return "USDC";
-        if (cToken == cDaiAddress) return "DAI";
-        if (cToken == cRepAddress) return "REP";
-        if (cToken == cWbtcAddress) return "BTC";
-        if (cToken == cBatAddress) return "BAT";
-        if (cToken == cZrxAddress) return "ZRX";
-        if (cToken == cSaiAddress) return "SAI";
-        if (cToken == cUsdtAddress) return "USDT";
+    function getOracleKey(address cToken) public view returns (bytes32) {
+        if (cToken == cEthAddress) return  symbolEth;
```

Comment:
I think I like this idea, what if we just went for using the cToken address for the view storage key , then we don't need this functino at all

Reply:
I like this idea. We can store prices as mapping from address => uint , when posting prices, map string to this address, and delete getOracleKey completely. 

@jflatow what do you think?

Reply:
So prices would be mapping cToken to price? I think since we have the string => price in the data contract its probably fine...but this was really a 'Rob constraint' originally so we need to ask him if he's ok with it